README.md

```diff
@@ -18,9 +19,10 @@ This role requires:
 
 The following OS are currently actively tested using kitchen:
 
-- Centos 6 & 7
-- Debian 7 & 8
-- Ubuntu 14.04
+- Centos 6 & 7 +
+- Debian 7 & 8 +
+- Ubuntu 14.04 +
+- Windows 2008R2 +
```

Comment:
in my test, all of the beats(+ansible) run with windows server 2008R2, win server 2019 is ok. because ansible for windows need PowerShell version 3.0 and .NET Framework 4.0 . ansible's doc [windows_setup.html#host-requirements](https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html#host-requirements), but windows server 2008 is not a good choice. suggest win server 2008R2.

Reply:
@jmlrt How about migrating tests to Molecule? It's more common for ansible testing.

Reply:
@tgadiev [ansible-beats](https://github.com/elastic/ansible-beats) and [ansible-elasticsearch](https://github.com/elastic/ansible-elasticsearch) are using Kitchen-CI and Serverspec for historical reasons, despite I would prefer using Molecule for that, we don't have enough workload dedicated to migrate our existing tests to Molecule for these 2 Ansible roles. Also we want to keep consistency so we won't migrate ansible-beats without ansible-elasticsearch.

As for Windows support, I'm not sure how migrating tests to Molecule would help.